### PR TITLE
updated node operator guide

### PR DESCRIPTION
**Pull request by shunsukew**

### CODE REVIEW

**Review by fiexer — APPROVED** [review]

File: docs/nodes/archive-node/binary.md [hunk 1]
```diff
@@ -49,12 +52,17 @@ To run a stable collator node, a **systemd service** has to be set and activated
 Create a service file
 
 ```sh
-sudo nano /etc/systemd/system/astar.service
+sudo vi /etc/systemd/system/astar.service
```
Comment (bLd75) [inline docs/nodes/archive-node/binary.md:55]: I would keep nano here, more accessible for everyone

File: docs/nodes/archive-node/index.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 ## Overview
 
-An **archive node** keeps all the past blocks. It plays a vital role on a blockchain: it connects users and dApps to the blockchain through WS and RPC endpoints. The public endpoints you can see in [network details](https://docs.astar.network/integration/network-details) are running archive nodes.
+An **archive node** keeps all the past blocks. It plays a vital role on a blockchain: it connects users and dApps to the blockchain through WS and RPC endpoints. The public endpoints you can see in [network details](/docs/quickstart/endpoints) are running archive nodes.
 
 **Dapp projects** need to run their own archive node to retrieve blockchain data they use in order not to rely on public infrastructure that will respond slower because of the large amount of users connected.
 
 :::note
-Be careful of the confusion with a **full node** that has a prunned database: a full node only keeps the past 256 blocks and uses much less storage space.
+Be careful of the confusion with a **full node** that has a prunned database: a full node only keeps the past configured number of blocks and uses much less storage space.
```
Comment (bLd75) [inline docs/nodes/archive-node/index.md:13]: add 

> (256 blocks by default)

File: docs/nodes/collator/requirements.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ To become a permissionless collator on one of our networks, you need to meet the
   <li>Meet hardware requirements</li>
 </ul>
 </p>
-<p>If your node stops producing blocks for 1 session, your node will be kicked out of the active set. Running a node with low performance can lead to skipping blocks which can lead to being kicked out of the active set.</p>
+<p>If your node stops producing blocks for 1 session, your node will be kicked out from the active set. Running a node with low performance can lead to skipping blocks which can lead to being kicked out of the active set.</p>
```
Comment (bLd75) [inline docs/nodes/collator/requirements.md:26]: And slashed 1% of the bonded funds

File: docs/nodes/collator/requirements.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ To become a permissionless collator on one of our networks, you need to meet the
...
```
Comment (bLd75) [inline docs/nodes/collator/requirements.md:36]: And slashed 1% of the bonded funds

File: docs/nodes/collator/secure_setup_guide/building_node.md [hunk 5]
```diff
@@ -157,10 +174,13 @@ To run a stable collator node, a **systemd service** has to be set and activated
 Create a service file
 
 ```
-sudo nano /etc/systemd/system/astar.service
+sudo vi /etc/systemd/system/astar.service
```
Comment (bLd75) [inline docs/nodes/collator/secure_setup_guide/building_node.md:177]: Same here I'd keep nano

**Review by bLd75 — APPROVED** [review]

**Pull request merged**